client/client.go

```diff
@@ -157,16 +180,19 @@ func (q Query) Result() Result {
 	return val.(Result)
```

Comment:
The undesired behaviour here will cause the system to panic or crash with wrong type inference?

Reply:
Yes, I said as much in the comment. I didn't change this code.

Reply:
👍